schema.graphql

```diff
@@ -1806,6 +1812,14 @@ scalar DateTime
 
 scalar Upload
 
+input KeywordsInput {
+  keywords: [String!]
```

Comment:
Based on your [implementation of resolver](https://github.com/thematters/matters-server/pull/2668/files#diff-92ae87dbbe12dd8451cca13dfcc694befc03f804ebe23b887187ee99223561e5R16), I think here should be `[ID!]`